login-workflow/src/contexts/AuthContext/provider.tsx

```diff
@@ -22,24 +22,24 @@ export const AuthContextProvider: React.FC<
     const { language, i18n = i18nInstance, errorConfig } = props;
```

Comment:
This is a little bizarre...not sure if it's causing your issue, but you're doing what looks like a double destructuring of the i18n prop. You set `i18nInstance` to the passed prop or the default auth instance...and then you destructure the `i18n` prop again on line 22. I don't think causes any errors, it's just very strange.

Reply:
App translations were breaking because app routes were part of auth routes( earlier change password was part of auth context this might be the reason for moving app routes inside auth). Fixed it now.